File: apps/ios/Barns/BarnsApp/Domain/Entities/GreeneryInfo.swift
import Foundation

/// Official, read-only basic information about a greenery type. Concise and
/// UI-friendly for a future basic-information screen.
///
/// Boundary: official shared content only. No user-owned state.
struct GreeneryInfo: Equatable, Identifiable, Sendable {
    let id: String
    let name: String
    let overview: String
    let difficulty: String
    let recommendedEnvironment: String
    let lightPreference: String
    let wateringOverview: String
    let maintenanceNotes: String
    let imageUrl: URL?
}

File: apps/android/app/src/main/java/com/barns/app/domain/model/GreeneryInfo.kt
package com.barns.app.domain.model

/**
 * Official, read-only basic information about a greenery type. Concise and
 * UI-friendly for a future basic-information screen.
 *
 * Boundary: official shared content only. No user-owned state.
 */
data class GreeneryInfo(
    val id: String,
    val name: String,
    val overview: String,
    val difficulty: String,
    val recommendedEnvironment: String,
    val lightPreference: String,
    val wateringOverview: String,
    val maintenanceNotes: String,
    val imageUrl: String?,
)
